specification/dsdl/dsdl.tex

```diff
@@ -132,30 +132,77 @@ \section{Syntax}\label{sec:dsdl_syntax}
 directives apply only to the part of the service type definition where they are defined,
 not crossing the boundary of the service response marker.
 
-\subsection{Attribute definition}
 
-Field definitions follow one of the below specified declaration patterns:
+\subsection{Constant expressions}\label{sec:dsdl_constant_expressions}
+A valid constant expression is either just a constant, or an expression that results in a constant.
+Such expression is given in infix notation,
+this means \texttt{A <op> B} for binary operators and \texttt{<op> A} for unary operators.
+A comprehensive table of legal operations on types is defined below,
+the order of the table also defines the order of precedence between operations.
+
+\begin{UAVCANSimpleTable}{Operators for constant expressions}{|c c c c c l|} \label{table:dsdl_const_operators}
+Presedence & Op & A & B & Res & Description \\
```

Comment:
`Precedence`

If you are using vscode, I highly recommend https://marketplace.visualstudio.com/items?itemName=ban.spellright

Reply:
Thanks, I will look into it.